sphinx/ext/intersphinx.py

```diff
@@ -229,6 +230,7 @@ def fetch_inventory_group(
                 logger.info(__('loading intersphinx inventory from %s...'), safe_inv_url)
                 try:
                     invdata = fetch_inventory(app, uri, inv)
+                    print(invdata)
```

Comment:
It's probably worth removing this (debug?) `print` statement?

Reply:
Oups I forgot about it!